ArduSub/GCS_Mavlink.cpp

```diff
@@ -256,6 +257,32 @@ NOINLINE void Sub::send_extended_status1(mavlink_channel_t chan)
         0, // comm drops in pkts,
         0, 0, 0, 0);
 
+    // FMUV2 support only
+#if CONFIG_HAL_BOARD == HAL_BOARD_PX4
```

Comment:
This only says it for the PX4 HAL, doesn't say anything about the board . and with hard-coded pin numbers you definitely want to check for the correct board. I guess that adding another macro to check if CONFIG_ARCH_BOARD_PX4FMU_V2 is defined is enough.